fixed typos (#322)
usebruno/bruno-docs: src/pages/open-api/importOAS.mdx

### Importing from a URL

> The import method for URL option is same as for URL but you need to select **OpenAPI V3 URL** as option and provide URL of OAS file which is publicly available. 

Reviewer: ```suggestion
Select **OpenAPI V3 URL** and provide the public URL of the OpenAPI file you're trying to import. 
```
